source/c.txt

```diff
@@ -75,6 +76,37 @@ Connect to MongoDB Atlas
 See `Advanced Connections <http://mongoc.org/libmongoc/current/advanced-connections.html>`__
 for more ways to connect.
 
+Versioned API
+-------------
+
+You can use the `Versioned API
+<http://docs.mongodb.com/v5.0/reference/versioned-api/>`__ feature
+starting with MongoDB server version 5.0 and C Driver version 1.18. When
+you use the Versioned API feature with an official MongoDB driver, you
```

Comment:
I think we need to mention that they must be using both Versioned API on the server and driver. You will likely encounter an error if you try to use Versioned API with the driver on a server version that doesn't support it.

Reply:
I think since the server and driver versions are specified in the first sentence, this sentence would impart enough information if it just read, "When you use the Versioned API feature, you..."